nosql-db/infinispan/src/main/java/io/quarkus/ts/infinispan/client/FirstCounterResource.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.ts.infinispan.client;
+
+import javax.ws.rs.Path;
+
+@Path("/first-counter")
+public class FirstCounterResource extends InfinispanCounterResource {
```

Comment:
I am not sure if this class is required or if we can just move the Path definition to `InfinispanCounterResource`